src/tox/config/cli/parser.py

```diff
@@ -162,7 +162,7 @@ def add_command(
             cmd,
             help=help_msg,
             aliases=aliases,
-            formatter_class=HelpFormatter,
+            formatter_class=argparse.RawTextHelpFormatter,
```

Comment:
You cannot do this, you're removing the entire HelpFormatter functionality :thinking:

Reply:
Ah yes, my bad, I should have edited `HelpFormatter` instead. I see you just did, thank you! 😄